packages/components/src/__actions__/Menu/v3/MenuHeader.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react"
+import { Header } from "react-aria-components"
+import styles from "./MenuHeader.module.scss"
+
+type MenuHeaderProps = {
+  children: any
+}
+
+/**
+ *
+ */
+export const MenuHeader = (props: MenuHeaderProps): JSX.Element => (
+  <Header className={styles.header} {...props} />
```

Comment:
forwardRef, but also it should possible to pass in `className` and merge it with our styles